profiler/profiler/nbproject/project.properties

```diff
@@ -20,6 +20,7 @@ is.autoload=true
 cp.extra=${tools.jar}
 
 javac.source=1.8
+bootclasspath.prepend=${build.dir}/disable-release
```

Comment:
I assume this is to disable setting "release" version as linking against com.sun classes happens.

Reply:
Right: 94aebf0